src/main/java/com/raisetech/api/sean/controller/PlayerExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.raisetech.api.sean.controller;
+
+import com.raisetech.api.sean.service.PlayerNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class PlayerExceptionHandler {
+
+    @ExceptionHandler(PlayerNotFoundException.class)
+    public ResponseEntity<String> handlePlayerNotFoundException(PlayerNotFoundException e) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(e.getMessage());
+    }
+
+    @ExceptionHandler(RuntimeException.class)
```

Comment:
RuntimeExceptionよりも上位のExceptionをハンドルするのがいいかもですね。
```suggestion
    @ExceptionHandler(Exception.class)
```

Reply:
https://docs.oracle.com/javase/jp/8/docs/api/java/lang/RuntimeException.html
<img width="422" alt="スクリーンショット 2023-10-17 16 29 02" src="https://github.com/Satoru-Oki/9th-topic-raisetech/assets/62045457/3ceb7b36-b910-4311-8913-1f81a139adca">

ただ、ThrowableだとErrorも拾っちゃうのでExceptionのほうがよいですね。

Reply:
なんにせよこのあたりのエラーのケアが発想としてあるのはすばらしい。

Reply:
これはいいエンジニアになるな。

Reply:
お褒めに預かり光栄です！・・・が受講生の皆さまのものやネットに出ているものをパクっただけでございます！
m(_  _)m
Runtimeについては何か軽いものぐらいのイメージでした。教えていただきありがとうございます。
RuntimeExceptionをExceptionに変更しています。
 `@ExceptionHandler(Exception.class)
    public ResponseEntity<String> handleException(Exception e) {`